tests/YearWeekTest.php

```diff
@@ -368,6 +369,23 @@ public function testGetFirstLastDay(int $year, int $week, string $firstDay, stri
         $this->assertIs(LocalDate::class, $lastDay, $yearWeek->getLastDay());
     }
 
+    /**
+     * @dataProvider providerGetFirstLastDay
+     *
+     * @param int $year
+     * @param int $week
+     * @param string $firstDay
+     * @param string $lastDay
+     * @return void
+     */
+    public function testToLocalDateRange(int $year, int $week, string $firstDay, string $lastDay): void
+    {
+        $yearWeek = YearWeek::of($year, $week);
+        $expectedDareRange = LocalDateRange::parse($firstDay . '/' . $lastDay);
+
+        $this->assertEquals($expectedDareRange, $yearWeek->toLocalDateRange());
```

Comment:
Could you please compare their string representations instead of loosely comparing objects?